src/MessengerPlugin.jsx

```diff
@@ -31,14 +34,25 @@ export default class MessengerPlugin extends Component {
     }
 
     initFacebookSDK() {
-        const {FB, appId} = this.props;
+        const {FB, appId, onRender, onClick, onNotYou} = this.props;
 
         if (FB) {
             FB.init({
                 appId,
                 xfbml: true,
                 version: 'v2.6'
             });
+
+            FB.Event.subscribe('send_to_messenger', function(e) {
+              if(onRender && e.event === "rendered") {
```

Comment:
a bit of formatting needed here :) the project includes an .esformatter config file to make sure it's consistent.